templates/test/details.html.ep

```diff
@@ -21,6 +21,7 @@
                     <td class="component">
                         <div>
                             %= link_to $module->{name} => url_for('src_step', stepid => 1, moduleid => $module->{name}, testid => $testid)
+                            <button onclick="showThumbnails('<%== $i %>')">+/-</button>
```

Comment:
sure , it's still WIP so for sure I am not suggesting to add this ugly button to our UI. Main purpose of this PR in current stage just get general feedback about idea